database/migrations/2022_04_06_000000_create_blog_article_table.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    public function up()
+    {
+        Schema::create('blog_articles', function (Blueprint $table) {
+            $table->id();
+            $table->string('title');
+            $table->string('author');
+            $table->unsignedBigInteger('category_id')->nullable();
+            $table->string('slug');
+            $table->string('layout')->nullable();
+            $table->unsignedInteger('footer_id')->nullable();
```

Comment:
If you're gonna use the `Publishable` interface it would be a good idea to add `$table->timestamp('published_at')->nullable();`

```suggestion
            $table->unsignedInteger('footer_id')->nullable();
            $table->timestamp('published_at')->nullable();
```